openfecwebapp/views.py

```diff
@@ -67,6 +68,17 @@ def render_legal_mur(mur):
         parent='legal'
     )
 
+def render_legal_ao_landing():
+    today = datetime.date.today()
+    ao_min_date = today - datetime.timedelta(weeks=26)
+    results = api_caller.load_legal_search_results(query='', query_type='advisory_opinions', ao_min_date=ao_min_date)
+    recent_aos=OrderedDict(sorted(results['advisory_opinions'].items(), key=lambda item: item, reverse=True))
```

Comment:
Thanks for explaining! I think I get it, and I'm pretty sure it's what we want. `api_caller.load_legal_search_results()` returns an `OrderedDict` for `results['advisory_opinions']` that looks like:

```
OrderedDict([
  ('2016-09', [{'category': 'Final Opinion',...}]),
  ('2016-07', [{'category': 'Final Opinion',...}])
])
```

Which we then want to loop through on the front-end. So it seems like it makes sense to just turn that into a new `OrderedDict` sorted by the first value of each pair.